exception ExampleError(int value);

class TestClass (int value) {
	static create = default;
	
	int add(int value) {
		throw ExampleError(1);
	}
}

void function() noexcept {
	(void) (TestClass(1) + 1);
}

// Entry point.
export int testEntryPoint(unused int argc, unused ubyte ** argv) {
	return 0;
}

